frontend/containers/SurplusContainer.vue

```diff
@@ -102,13 +136,27 @@ export default Vue.extend({
             refreshWallet: 'fetchWalletBalances',
             disconnectWallet: 'disconnect',
         }),
+        authorizeWallet() {
+            this.$store.dispatch('authorizations/authorizeWallet');
+        },
+        withdrawAllDaiFromVat() {
+            this.$store.dispatch('wallet/withdrawAllCollateralFromVat', 'DAI');
+        },
```

Comment:
here you want to withdraw DAI,  which i don't think is a collateral. Please verify this call. I would expect that `withdrawFromVAT` would be called.